src/file-upload/file-item-edit.tsx

```diff
@@ -138,11 +148,13 @@ export const FileItemEdit = ({
                     type="button"
                     styleType="secondary"
                     data-testid={`${id}-cancel-button`}
-                    onClick={handleCancel}
+                    onClick={onCancel}
                 >
                     Cancel
                 </ActionButton>
             </ActionButtonsSection>
         </Item>
     );
 };
+
+export const FileItemEdit = memo(Component);
```

Comment:
what's the intention behind memoizing the list items? specifically for `FileItemEdit`, the memoization does not help as a new `fileItem` object reference is created on each render

Reply:
I think this was done previously when I was testing. Will revert.